src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/model/entity/Visit.kt

```diff
@@ -99,12 +99,10 @@ data class Visit(
 
   @Column
   var reference = _reference
-    private set
```

Comment:
had to remove this private set as open classes cannot have private set methods it seems.